### ISSUE TITLE
i got flask error

### ISSUE THREAD

**Issue by HEXtoDEC** [Sanster/lama-cleaner#93]
http://127.0.0.1:8080//flaskwebgui-keep-server-alive

GET 404

```shell
2022-10-21 13:36:12.346 | INFO     | lama_cleaner.model.base:_pad_forward:53 - final forward pad size: (512, 736, 3)
127.0.0.1 - - [21/Oct/2022 13:36:14] "GET /model HTTP/1.1" 200 -
  0%|                                                                                                                                                                 | 0/51 [00:00<?, ?it/s]
[2022-10-21 13:36:16,034] ERROR in app: Exception on /inpaint [POST]
Traceback (most recent call last):
  File "C:\Users\UserName\miniconda3\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\UserName\miniconda3\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\UserName\miniconda3\lib\site-packages\flask_cors\extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "C:\Users\UserName\miniconda3\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\UserName\miniconda3\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\UserName\miniconda3\lib\site-packages\lama_cleaner\server.py", line 142, in process
    res_np_img = model(image, mask, config)
  File "C:\Users\UserName\miniconda3\lib\site-packages\lama_cleaner\model_manager.py", line 34, in __call__
    return self.model(image, mask, config)
  File "C:\Users\UserName\miniconda3\lib\site-packages\torch\autograd\grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\UserName\miniconda3\lib\site-packages\lama_cleaner\model\sd.py", line 186, in __call__
    inpaint_result = self._pad_forward(image, mask, config)
  File "C:\Users\UserName\miniconda3\lib\site-packages\lama_cleaner\model\base.py", line 55, in _pad_forward
    result = self.forward(pad_image, pad_mask, config)
  File "C:\Users\UserName\miniconda3\lib\site-packages\torch\amp\autocast_mode.py", line 12, in decorate_autocast
    return func(*args, **kwargs)
  File "C:\Users\UserName\miniconda3\lib\site-packages\lama_cleaner\model\sd.py", line 137, in forward
    output = self.model(
  File "C:\Users\UserName\miniconda3\lib\site-packages\torch\autograd\grad_mode.py", line 27, in decorate_context
    return func(*args, **kwargs)
  File "C:\Users\UserName\miniconda3\lib\site-packages\diffusers\pipelines\stable_diffusion\pipeline_stable_diffusion_inpaint.py", line 387, in __call__
    latent_model_input = torch.cat([latent_model_input, mask, masked_image_latents], dim=1)
RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 64 but got size 92 for tensor number 2 in the list.
127.0.0.1 - - [21/Oct/2022 13:36:16] "POST /inpaint HTTP/1.1" 500 -
````

**Comment by Sanster** [comment]
You can ignore the 404 error, the real problem causing the 500 error is fixed in `0.24.2`, please update the version

**Comment by HEXtoDEC** [comment]
> You can ignore the 404 error, the real problem causing the 500 error is fixed in `0.24.2`, please update the version

it's work!
